@re-/assert/test/metaTests/templateForBenches.ts

```diff
@@ -0,0 +1,16 @@
+import { bench } from "../../src/index.js"
+
+//median
+bench("bench call single stat", () => {
```

Comment:
One thing I should've definitely said earlier is that you can pass an `until` option where you can specify how long or for how many iterations the bench runs. In this case, we can just do 2 iterations or something so we wouldn't have to wait.